Sources/KlaviyoForms/InAppForms/IAFPresentationManager.swift

```diff
@@ -146,6 +165,11 @@ class IAFPresentationManager {
             Logger.webViewLogger.info("Handling '\(event.rawValue, privacy: .public)' form lifecycle event")
         }
         switch event {
+        case .formsDataLoaded:
+            // Replay buffered events immediately now that forms data is ready
+            Task {
+                await replayPendingEvents()
```

Comment:
This is the main path i.e when IAF JS module emits `formsDataLoaded` we will replay the events that were queued/buffered.